### PR TITLE
Task SWE 1150 write IdeaScreen

### PR DESCRIPTION
**Pull request by Olexandr25**
Screenshot 2022-07-19 at 16 33 21
Screenshot 2022-07-19 at 16 33 48

### CODE REVIEW

File: src/app/components/layouts/ScreenSlider/ScreenSlider.js [hunk 1]
```diff
@@ -55,10 +54,14 @@ const ScreenSlider = forwardRef((props, ref) => {
             }
             ref={ref}
             {...rest}>
+            {/* <ColorScreen />
             <ColorScreen />
             <ColorScreen />
-            <ColorScreen />
-            <ColorScreen />
+            <ColorScreen /> */}
+            <IdeaScreen />
```
Comment (5P4C3GR33D) [inline src/app/components/layouts/ScreenSlider/ScreenSlider.js:61]: What's going here?..

File: src/app/components/layouts/ScreenSlider/ScreenSlider.js [hunk 2]
```diff
@@ -1,8 +1,7 @@
 import { Button, Icon, Slider, Title } from 'app/components'
+import { ColorScreen, IdeaScreen } from 'app/screens'
 import React, { forwardRef, useState } from 'react'
 
-import { ColorScreen } from 'app/screens'
-
 const ScreenSlider = forwardRef((props, ref) => {
   const {
```
Comment (5P4C3GR33D) [inline src/app/components/layouts/ScreenSlider/ScreenSlider.js:6]: Add propTypes.

File: src/app/components/layouts/ScreenSlider/ScreenSlider.js (hunk 1 above)
Comment (Olexandr25) [reply src/app/components/layouts/ScreenSlider/ScreenSlider.js:61]: Component, which has different screens

File: src/app/components/layouts/ScreenSlider/ScreenSlider.js (hunk 2 above)
Comment (Olexandr25) [reply src/app/components/layouts/ScreenSlider/ScreenSlider.js:6]: If I knew, what is there inside this props, I would write it in the bottom

**Review by 5P4C3GR33D — APPROVED** [review]

**Pull request merged**